sources/main.cpp

```diff
@@ -96,31 +103,45 @@ main(int argc, char* argv[]) {
     dfl::Context context(def, config);
 
     if (!context.process()) {
+      LOG(info) << MESS(InitEnd, timerInit.elapsed()) << LOG_ENDL;
       return EXIT_FAILURE;
     }
+    LOG(info) << MESS(InitEnd, timerInit.elapsed()) << LOG_ENDL;
 
+    boost::timer timerFiles;
     context.exportOutputs();
-    context.execute();
-
     dfl::common::Log::init(options, config.outputDir(), continuePreviousLogFile);
+    LOG(info) << MESS(FilesEnd, timerFiles.elapsed()) << LOG_ENDL;
+
+    boost::timer timerSimu;
+    context.execute();
 
-    LOG(info) << MESS(SimulationEnded, context.basename()) << LOG_ENDL;
+    LOG(info) << " ============================================================ " << LOG_ENDL;
+    LOG(info) << MESS(SimulationEnded, context.basename(), timerSimu.elapsed()) << LOG_ENDL;
```

Comment:
Maybe the last that should appear is the total time, not the simulation time. I was confused the fist time I read the logs